.github/workflows/pythonpackage.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           SCRAPER_API_KEY: ${{ secrets.SCRAPER_API_KEY }}
         run: |
           curl --data-binary @.github/.codecov.yml https://codecov.io/validate | head -n 1
-          coverage run -m unittest -v test_module.py
+          coverage run -m unittest -v test_module.TestScholarly
```

Comment:
Luminati is not being set up successfully with the credentials available in the repo and hence testing only the `TestScholarly` class